deploy/firesim

```diff
@@ -149,6 +149,22 @@ def buildafi(globalbuildconf):
     execute(aws_build, globalbuildconf, hosts=globalbuildconf.get_build_instance_ips())
 
 
+def tar2afi(globalbuildconf):
+    """
+    Starting from the tarball generated by Vivado generate an AFI
+
+    Iterate the globalbuildconf.get_builds_list() using a multiprocessing.Pool
+    running the logic for submitting the tarball to `aws ec2 create-fpga-image`,
+    wait for afi to be available and copy to all known regions
+
+    NOTE: this is only useful if you also pass the --launchtime argument to firesim
+    and specify an already existing launchtime for a previous, tarball generation
+    """
+
+    for buildconf in globalbuildconf.get_builds_list():
+        execute(aws_create_afi, globalbuildconf, buildconf, hosts=['localhost'])
```

Comment:
This could be improved.  As it is now, all of the configs will execute serially.  I don't see how to get Fabric to execute them in parallel on localhost.  I don't really want to have to spin up a buildfarm only for the purposes of then running things in parallel on localhost.  I thought about using multiprocessing or threading to do this in parallel but it seems ugly to import another subprocess execution library.

I think this is useful even with them running in series.

Thoughts?

Reply:
I could've sworn i ran stuff in parallel on the localhost in the past. Maybe you can work around this by specifying repeated instances of the localhost in the worker pool?